components/chart/src/range/range.ts

```diff
@@ -84,20 +84,102 @@ export class RangeStateChangedEvent {
   },
 })
 export class DtChartRange implements AfterViewInit, OnDestroy {
+  /**
+   * Aria label for the close button in the overlay.
+   * @deprecated `aria-label-close` does not conform with accessibility standards.
+   * Please use `ariaLabelClose` input instead.
+   * @breaking-change Will be removed in version 7.0.0
+   */
+  @Input('aria-label-close')
+  get depAriaLabelClose(): string {
+    return this._ariaLabelClose;
+  }
+  set depAriaLabelClose(value: string) {
+    this._ariaLabelClose = value;
+  }
+
   /** Aria label for the close button in the overlay */
-  @Input('aria-label-close') ariaLabelClose = ARIA_DEFAULT_CLOSE_LABEL;
+  @Input()
+  get ariaLabelClose(): string {
+    return this._ariaLabelClose;
+  }
+  set ariaLabelClose(value: string) {
+    this._ariaLabelClose = value;
+  }
+  /** @internal Aria label value for the close button of the overlay */
+  _ariaLabelClose = ARIA_DEFAULT_CLOSE_LABEL;
 
-  /** Aria label for the left handle */
+  /**
+   * Aria label for the left handle
+   * @deprecated `aria-label-left-handle` does not conform with accessibility standards.
+   * Please use `ariaLabelLeftHandle` input instead.
+   * @breaking-change Will be removed in version 7.0.0
+   */
   @Input('aria-label-left-handle')
-  ariaLabelLeftHandle = ARIA_DEFAULT_LEFT_HANDLE_LABEL;
+  get depAriaLabelLeftHandle(): string {
```

Comment:
They are needed, because the name of the input and the property change, so we need to keep consistency with the property and input and keep them working for at least 2 versions.